spidy/tests/int_test.py

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env python
+
+import unittest
+import rostest
+import rospy
+from std_msgs.msg import Float64
+
+bat=0
+class MyTestCase(unittest.TestCase):
+
+    def callback(self, data):
+        bat=data.data
+
+
+    def test_whatever(self):
+        rospy.init_node('listener', anonymous=True)
+        rospy.Subscriber("chatter", Float64, self.callback)
+        self.assertTrue(bat<100)
```

Comment:
as i see it tests nothing because it is just comparing zero with hundred. And i wrote about moving in the task.

Reply:
has been modified to much better test technique which i compare the value i send for movement with the value that the robot already move